src/dmae/main_pretrain.py

```diff
@@ -346,6 +346,12 @@ def main(args):
     total_time = time.time() - start_time
     total_time_str = str(datetime.timedelta(seconds=int(total_time)))
     print("Training time {}".format(total_time_str))
+    
+    ckpt_dir = os.path.dirname(checkpoint_path)
```

Comment:
Why not just store one checkpoint.pth file and overwrite it every epoch (or depending upon improvement in val loss) instead of storing one per epoch?

Reply:
This is a little brittle since the training workflow can fail before we reach the delete step.

Reply:
Perhaps we should consider keeping it. 
We do not get the validation performance, so we there's no signal to decide on during pretraining. I think it is actually worthwhile to keep some history, I've already significantly cut down on the #ckpts, I'd imagine by an order of magnitude. There have been times when the most recent ckpt is messed up, and i've had to rollback to an older one.